hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This patch fixes a bug where only interactively-generated values (via ``data.draw``) would be reported in the ``arguments`` field of our :doc:`observability output <observability>`. Now, all values are reported.
```

Comment:
It'd be nice to identify "a bug where ... since version $version" if you have that to hand.

Reply:
wasn't too bad to bisect https://github.com/HypothesisWorks/hypothesis/pull/3935